README.rst

```diff
@@ -98,10 +98,11 @@ For licensing information, please view ``docker/README.md``.
 Build PennyLane-Lightning-GPU with multi-node/multi-gpu support
 ---------------------------------------------------------------
 
-Use of PennyLane-Lightning-GPU with multi-node/multi-gpu support also requires explicit installation of the NVIDIA cuQuantum SDK, ``mpi4py`` 
-and `CUDA-aware MPI` (Message Passing Interface). `CUDA-aware MPI` allows data exchange between GPU memory spaces of different nodes
-without the need for CPU -mediated transfers. Both `MPICH` and `OpenMPI` libraries are supported, provided they are compiled with CUDA support.
-The NVIDIA cuQuantum library and MPI library directory may be provided on the ``LD_LIBRARY_PATH`` environment variable, or the NVIDIA cuQuantum SDK and ``mpi4py`` 
+Use of PennyLane-Lightning-GPU with multi-node/multi-gpu support also requires explicit installation of the ``NVIDIA cuQuantum SDK`` (current supported 
+cuQuantum version: `cuquantum-cu11 <https://pypi.org/project/cuquantum-cu11/>`_), ``mpi4py`` and ``CUDA-aware MPI`` (Message Passing Interface). 
+``CUDA-aware MPI`` allows data exchange between GPU memory spaces of different nodes without the need for CPU-mediated transfers. Both ``MPICH`` 
+and ``OpenMPI`` libraries are supported, provided they are compiled with CUDA support.
+The NVIDIA cuQuantum library and MPI library directory may be provided on the ``LD_LIBRARY_PATH`` environment variable, or the ``NVIDIA cuQuantum SDK`` and ``mpi4py``. 
```

Comment:
I'm a little unclear about this sentence.

Reply:
Any suggestions on the updated one?